kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -1034,29 +1025,21 @@ extension [A, E, Ctx](effect: A < (Abort[E] & Async & Ctx))
       *   A computation that produces the result of this computation
       */
     @targetName("zipLeftPar")
-    inline def <&[A1, E1, Ctx1](next: A1 < (Abort[E1] & Async & Ctx1))(
+    def <&[A1, E1, S1](next: A1 < (Abort[E1] & Async & S1))(
```

Comment:
We should migrate these methods to use `Async.parallel` so `Isolate.Stateful` is supported. I can work on this with the `Async` changes I'm planning